Tiltfile

```diff
@@ -215,11 +218,6 @@ def flavors():
     cfg = config.parse()
     worker_templates = cfg.get('templates-to-run', [])
 
-    substitutions = settings.get("kustomize_substitutions", {})
```

Comment:
we need to leave this line since substitutions is used later on line 223

`Tiltfile:223:8: undefined: substitutions`